libpod/events/events.go

```diff
@@ -79,7 +81,7 @@ func (e *Event) ToHumanReadable(truncate bool) string {
 	}
 	switch e.Type {
```

Comment:
Can you do this once before the sitch and then use it in each of the case statements?

timUnix:= time.Unix(0, e.TimeNano)

Also why use this rather then just using

e.Time?